admin/tabs/general/roles/details/members/RoleMembersModel.ts

```diff
@@ -47,14 +47,9 @@ export class RoleMembersModel extends HoistModel {
     }
 
     @computed
-    get directCounts(): Record<RoleMemberType, number> {
+    get inheritedRolesCount(): Record<string, number> {
```

Comment:
One change - I would change the return type to `Record<'ROLE', number>` since that's the only key that will ever exist for this object.